### DISCUSSION TITLE
Anyway to log into Google from it?

### DISCUSSION THREAD

**Discussion by pO0P3R125** [Voltstro-Studios/UnityWebBrowser#160] [Q&A]
This is not so much of an issue but more of a question, as the title implies I want it to login into a google account, is this possible through unity services? or OAuth2?

Thanks in advance

**Comment by Voltstro** [comment]
Google straight up blocks all logins requests from CEF.

If you're trying to get details of a user using Google OAuth, you are better off just opening the user's normal browser.

**Reply by pO0P3R125** [reply]
Oh, how would I go around doing that? Am a bit of a newbie in this kind of tech so I don't have much idea of it

**Reply by Voltstro** [reply]
I am not really gonna give a full on answer, as this has nothing to do with this project, plus it up to you to do the full research. But the general idea is that you take the User to Google to sign in with your app, User signs in, Google sends the User back to your app, and your app has an access token to call Google APIs on behalf of a user. Google has [docs on this whole process](https://developers.google.com/identity/protocols/oauth2).

Unity does have the [`Application.OpenURL`](https://docs.unity3d.com/ScriptReference/Application.OpenURL.html) method, allowing you to open the Google OAuth URL in the user's normal browser.
